API.md

```diff
@@ -751,6 +751,23 @@ Defines the fit mode (default zoom level) of the viewer.
 />
 ```
 
+#### fitPolicy
+
+Defines the fit mode (default zoom level) of the viewer.
+Parameters:
+
+Mode | Value | Description
+--- | --- | ---
+fit both or fitPage (default) | 0 | fits the whole page
```

Comment:
i don't understand what is "fit both or fitPage"

Reply:
it's like it fits both the height and width of the page or more like fits the page

Reply:
it's confusing to read, just do "fit page (default)"